pkg/gardenlet/controller/controllerinstallation/controllerinstallation/reconciler.go

```diff
@@ -40,6 +40,7 @@ import (
 	v1beta1helper "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
 	resourcesv1alpha1 "github.com/gardener/gardener/pkg/apis/resources/v1alpha1"
 	"github.com/gardener/gardener/pkg/client/kubernetes"
+	ctrlregistseeds "github.com/gardener/gardener/pkg/controllermanager/controller/controllerregistration/seed"
```

Comment:
```suggestion
	seedcontroller "github.com/gardener/gardener/pkg/controllermanager/controller/controllerregistration/seed"
```

Reply:
Also think about moving the constants to a common place, maybe `v1beta1constants` to avoid the newly introduced dependencies.